v2.1/export.md

```diff
@@ -0,0 +1,159 @@
+---
+title: EXPORT
+summary: Export tabular data from CockroachDB cluster in CSV format.
+toc: false
+---
+
+The `EXPORT` [statement](sql-statements.html) exports tabular data to CSV files.
```

Comment:
Should we say a little more here about what it is before diving into details?
like, that it exports the results of arbitratry SELECT statements or that it does it in a distributed process? That later point is what sets it apart from `sql --format=csv > myfile.csv`.